base/download.jl

```diff
@@ -4,7 +4,7 @@
 
 if Sys.iswindows()
     function download(url::AbstractString, filename::AbstractString)
-        ps = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
+        ps = joinpath(ENV["WINDIR"], "System32\\WindowsPowerShell\\v1.0\\powershell.exe")
```

Comment:
use `get(ENV, "WINDIR", "C:\\Windows")`

Reply:
I guess it's safer but WINDIR is defined by the system